pyogrio/tests/test_core.py

```diff
@@ -138,22 +138,27 @@ def test_read_bounds_bbox(naturalearth_lowres_all_ext):
     assert fids.shape == (0,)
     assert bounds.shape == (4, 0)
 
-    fids, bounds = read_bounds(naturalearth_lowres_all_ext, bbox=(-140, 20, -100, 45))
+    fids, bounds = read_bounds(naturalearth_lowres_all_ext, bbox=(-85, 8, -80, 10))
 
     assert fids.shape == (2,)
     if naturalearth_lowres_all_ext.suffix == ".gpkg":
+        # GPKG results are in opposite order
```

Comment:
For fgb we use `spatial_index=False` to avoid this ordering issue. The same might help for geopackage as well?